tools/bridge/tests/test_event_fetcher.py

```diff
@@ -140,6 +153,28 @@ def test_instantiate_event_fetcher_with_negative_max_reorg_depth(
             event_argument_filter=transfer_event_argument_filter,
             event_queue=transfer_event_queue,
             max_reorg_depth=-1,
+            start_block_number=foreign_chain_event_fetch_start_block_number,
+        )
+
+
+def test_instantiate_event_fetcher_with_negative_start_block_number(
```

Comment:
We seem to have a lot of tests like this which are very verbose. Suggestion: introduce a fixture `event_fetcher_init_kwargs` that returns (valid) key word arguments for `__init__` and then change only the relevant argument in the test. This would make the test much shorter:

```Py
with pytest.raises(ValueError):
    return EventFetcher(**assoc(event_fetcher_kwargs, "start_block_number", -1)
```

Reply:
Nice idea. Due to black we can save one more line per test case if we don't use `assoc`, but `**{**event_fetcher_kwargs, "start_block_number": -1}`.

Reply:
This also makes modifications for tests at the bottom with "special" cases cleaner.

Reply:
:heavy_check_mark: